README.md

```diff
@@ -1,50 +1,56 @@
 # Music Player - Python
 
-A `Music Player` application that allows you to listen to music files stored in the computer's internal or external memory.
+A `Music Player` application that allows you to listen to musics that stored in the computer's internal or external memory..
 
-![Table Image](images/sc1.png)
+![Table Image] (images/sc1.png)
 
-## Demo - How can you use my Music Player?
+## Demo - How you can use my Music Player?
 
 ### 1. Install all the dependencies:
 - <a target="_blank" href="https://www.python.org/ftp/python/3.11.1/python-3.11.1-amd64.exe">Install Python</a>
 
 - <a target="_blank" href="https://docs.python.org/3/library/tkinter.html">Tkinter</a>
     ```
-    pip install tkinter
+    Pip install tkinter
```

Comment:
In command line interface "pip" will be preferable rather "Pip"
So change all the "Pip" with "pip"